### DISCUSSION TITLE
New device added

### DISCUSSION THREAD

**Discussion by jetpuf** [iklein99/homebridge-smartthings#119] [General]
So I’ve added a new device to SmartThings, I rebooted Hoobs and the plug-in pulled it down correctly, how do I get HomeKit to refresh the devices to see the new equipment?

**Comment by iklein99** [comment]
Restart homebridge.  If it isn’t showing up, then maybe the device isn’t supported.

**Comment by jetpuf** [comment]
I have several of the same device type already in the system. Contact sensors and a smart plug that I re-joined to move from Groovy to Edge.On Jan 12, 2023, at 8:14 AM, iklein99 ***@***.***> wrote:﻿
Restart homebridge.  If it isn’t showing up, then maybe the device isn’t supported.
